packages/automated-dispute/src/services/eboActor.ts

```diff
@@ -421,13 +421,28 @@ export class EboActor {
         try {
             this.logger.info(`Settling dispute ${dispute.id}...`);
 
-            // OPTIMIZE: check for pledges to potentially save the ShouldBeEscalated error
+            const escalationData = await this.protocolProvider.read.getEscalation(request.id);
 
-            await this.protocolProvider.settleDispute(
-                request.prophetData,
-                response.prophetData,
-                dispute.prophetData,
-            );
+            const amountFor = escalationData.amountOfPledgesForDispute;
+            const amountAgainst = escalationData.amountOfPledgesAgainstDispute;
+
+            if (amountFor > amountAgainst) {
+                await this.protocolProvider.settleDispute(
+                    request.prophetData,
+                    response.prophetData,
+                    dispute.prophetData,
+                );
+
+                this.logger.info(`Dispute ${dispute.id} settled.`);
+            } else if (amountFor <= amountAgainst) {
```

Comment:
Any particular reason to be using less than and gte?
```suggestion
            if (amountFor !=== amountAgainst) {
                await this.protocolProvider.settleDispute(
                    request.prophetData,
                    response.prophetData,
                    dispute.prophetData,
                );

                this.logger.info(`Dispute ${dispute.id} settled.`);
            } else if (amountFor === amountAgainst) {
```